fileparse/UnitTest1.cs

```diff
@@ -31,11 +31,11 @@ public class UnitTest1 {
                 { @"C:\work\r.txt", true },
                 { @"C:\abc\def\r.txt.txt", false },
                 { @"C:\Program Files\<>*&^%$#@!.txt", false },
-                { @"C:\\work~\\r.txt~", false }
+                { @"C:\work~\r.txt~", false }
          };
-         foreach (var testCase in sTests) {
-            bool parseResult = FileParse.FileNameParse (testCase.Key, out _);
-            Assert.AreEqual (testCase.Value, parseResult);
+         foreach (var testCase in tests) {

```

Comment:
Can implement like this:
var fileParse = new FileParse ();
foreach (var testCase in tests) {
            bool parseResult = fileParse.FileNameParse (testCase.Key, out _);
            Assert.AreEqual (testCase.Value, parseResult);
         }